### ISSUE TITLE
Issue 4724: URLs with Copyright Symbols Detected as Copyright Statements

### ISSUE THREAD

**Issue by abraemer** [mstykow/scancode-rust#267]
ScanCode incorrectly detects URLs containing `(c)` characters as copyright statements.

## Key Details

- **Example File**: `urls.10K` from klauspost/compress v1.11.2 testdata
- **False Positive Example**:

  ```json
  {
    "copyright": "(c) /axove+xose/7,-1,0,B/frameset&F axuntanza&1,,3 http://biblio.cesga.es:81/search",
    "start_line": 4569
  }
  ```

- **Root Cause**: The `(c)` pattern in URLs triggers copyright detection
- **Reproduction**: `scancode --json-pp - --copyright urls.10K.txt`

## Proposed Fix

Improve context awareness in copyright detection to:

- Check if `(c)` appears within a URL pattern
- Consider surrounding context (URL structure, lack of year/author)
- Add URL patterns to junk/exclusion rules

## DIFFICULTY: MEDIUM

- Clear bug with reproduction steps
- Requires enhancement to copyright detection context awareness
- Need to avoid breaking legitimate `(c)` copyright statements

Ref: #4724
